tests/typecheck/fields/test_related.yml

```diff
@@ -649,10 +649,12 @@
         -   path: myapp/models.py
             content: |
                 from django.db import models
+                from django.db.models.manager import BaseManager
                 class TransactionQuerySet(models.QuerySet):
                     pass
+                TransactionManager = BaseManager.from_queryset(TransactionQuerySet)
                 class Transaction(models.Model):
-                    objects = TransactionQuerySet.as_manager()
```

Comment:
This is also important. Can we still be sure that this example works just fine?

Reply:
As far as I am aware there's no support for .as_manager, it'd basically need a corresponding approach to what .from_queryset has gotten. (Which I don't think exists?)

This triggered the newly introduced error, as the dynamically created type that .as_manager produces can't be found during type checking.

But I don't think we're introducing anything new here other than highlighting that .as_manager doesn't work. With an error

Reply:
Hm, actually I think we are improving here. Since one can now ignore the error and get a manager type, instead of a silent Any.

Reply:
Let's add an explicit test case for this then! 👍